tests/unit/src/test/scala/tests/CodeActionLspSuite.scala

```diff
@@ -26,6 +27,279 @@ object CodeActionLspSuite extends BaseLspSuite("codeAction") {
        |""".stripMargin
   )
 
+  check(
+    "use-named-arguments-basic",
```

Comment:
since these tests are going to grow for each code action, maybe we can split these into:

- `BaseCodeActionLspSuite` (which simply defines the `check` method)
- `ImportMissingSymbolLspSuite`
- `UseNamedArgumentsLspSuite`

This should have the benefits of:
- keeping the file length manageable
- keeping the suite more focused (you can run the entire suite when iterating on the specific action)
- removing the prefix from the test name (`basic`, instead of `use-named-arguments-basic`)

Reply:
Yes, I was thinking that as well.